packages/blobs/src/server.ts

```diff
@@ -318,10 +317,10 @@ export class BlobsServer {
       // happen if multiple clients try to write to the same key at the same
       // time. To prevent this, we write to a temporary file first and then
       // atomically move it to its final destination.
-      const tempPath = join(tmpdir(), Math.random().toString())
+      const tempPath = join(dirname(dataPath), '.' + Math.random().toString())
```

Comment:
nit: Shall we remove the `0.` from the directory name? I can't think of a specific case where that might become an issue, but feels a bit cleaner to get rid of it.

```suggestion
      const tempPath = join(dirname(dataPath), `.${Math.random().toString().slice(2)}`)
```

Reply:
we could also just use `uuid`. we already have it in the dependency tree (direct dep of `@netlify/dev-utils`).